sdk/dotnet/Pulumi.Automation.Tests/EventLogWatcherTests.cs

```diff
@@ -0,0 +1,100 @@
+namespace Pulumi.Automation.Tests
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.IO;
+    using System;
+    using Pulumi.Automation.Events;
+
+    using Xunit;
```

Comment:
These usings can be moved before the `namespace` declaration per typical C# code-style and a copy right comment can be added as the first line.

```suggestion
// Copyright 2016-2021, Pulumi Corporation

using System.Collections.Generic;
using System.Threading;
using System.Threading.Tasks;
using System.IO;
using System;
using Pulumi.Automation.Events;
using Xunit;

namespace Pulumi.Automation.Tests
{
```